### PR TITLE
Reverse array_find

### PR DESCRIPTION
**Pull request by m-mohr**
Implements #269

### CODE REVIEW

File: array_find.json [hunk 1]
```diff
@@ -45,9 +54,21 @@
...
+            "arguments": {
+                "data": [
+                    1,
+                    2,
+                    3
+                ],
+                "value": 3,
+                "reverse": true
+            },
+            "returns": 2
```
[the remark this answers is not in the document]
Comment (soxofaan) [reply array_find.json:75]: I think you swapped value and returns in this updated example

should be value=2 and returns=3

**Review by soxofaan — APPROVED** [review]

**Comment by soxofaan** [comment]
FYI: I approved, except for the note about swapping value and returns in the example

File: array_find.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-mohr) [reply array_find.json:75]: Or I forgot to take the reverse into account.

[the remark this answers is not in the document]
Comment (m-mohr) [reply array_find.json]: Fixed.

**Pull request merged**